### PR TITLE
x11-misc/idesk: bump to new versions

### PR DESCRIPTION
**Pull request by scardracs**
Drop gtk+2
Closes: https://bugs.gentoo.org/742746

Package-Manager: Portage-3.0.18, Repoman-3.0.3
Signed-off-by: Marco Scardovi <marco@scardovi.com>

### CODE REVIEW

File: x11-misc/idesk/idesk-1.0.ebuild [hunk 1]
```diff
@@ -0,0 +1,40 @@
+# Copyright 1999-2021 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+DESCRIPTION="Utility to place icons on the root window"
+HOMEPAGE="https://github.com/antonialoytorrens/idesk/"
+SRC_URI="https://github.com/antonialoytorrens/idesk/releases/download/v1/idesk-v1.tar.gz -> ${P}.tar.gz"
```
Comment (winterheart) [inline x11-misc/idesk/idesk-1.0.ebuild:8]: You need either to use ver_cut here or rename ebuild to idesk-1.ebuild and use here ${PV} as version substitute.

File: x11-misc/idesk/idesk-1.0.ebuild [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+	media-libs/imlib2[X]
+	media-libs/libart_lgpl
+	x11-libs/gtk+:3
+	x11-libs/pango
+	x11-libs/startup-notification
+"
+
+DEPEND="
+	${RDEPEND}
+	virtual/pkgconfig
```
Comment (winterheart) [inline x11-misc/idesk/idesk-1.0.ebuild:28]: virtual/pkgconfig should be in BDEPEND for EAPI=7

File: x11-misc/idesk/idesk-1.0.ebuild (hunk 1 above)
Comment (scardracs) [reply x11-misc/idesk/idesk-1.0.ebuild:8]: What is ver_cut? I have not understand that thing yet

File: x11-misc/idesk/idesk-1.0.ebuild (hunk 2 above)
Comment (scardracs) [reply x11-misc/idesk/idesk-1.0.ebuild:28]: Fixed

File: x11-misc/idesk/idesk-1.0.ebuild (hunk 1 above)
Comment (scardracs) [reply x11-misc/idesk/idesk-1.0.ebuild:8]: I have also seen here: https://devmanual.gentoo.org/ebuild-writing/variables/index.html

Comment (scardracs) [reply x11-misc/idesk/idesk-1.0.ebuild:8]: Fixed

**Comment by scardracs** [comment]
@winterheart can you check and say if everything is ok now?

File: x11-misc/idesk/idesk-1.ebuild [hunk 3]
```diff
@@ -0,0 +1,45 @@
...
+	media-libs/freetype
+	media-libs/imlib2[X]
+	media-libs/libart_lgpl
+	x11-libs/gtk+:3
+	x11-libs/pango
+	x11-libs/startup-notification
+"
+
+BDEPEND="
+	${RDEPEND}
```
Comment (winterheart) [inline x11-misc/idesk/idesk-1.ebuild:30]: You need to remove ${RDEPEND} from BDEPEND:
```
BDEPEND="virtual/pkgconfig"
```

Comment (scardracs) [reply x11-misc/idesk/idesk-1.ebuild]: Fixed

**Comment by scardracs** [comment]
@winterheart is it ok for you now?

**Review by juippis — APPROVED** [review]
Wonder if there's an easy way to update this:
```
# idesk --help
Idesk 0.7.5
==============
Default Prefix /usr
==============
Further documentation available at: http://idesk.sourceforge.net
```
Also, 
```
/usr/share/idesk/README
/usr/share/doc/idesk-1/README.bz2
```
are duplicates.

**Pull request closed without merging**